collection-renderer-lib/src/cmr/collection_renderer/services/collection_renderer.clj

```diff
@@ -32,6 +42,19 @@
     (.eval jruby (io/reader bootstrap-rb))
     jruby))
 
+(defn- get-preview-gem-umm-version
+  "Get the UMM schema version that is defined in preview gem."
+  []
+  (try
+    (-> preview-gem-umm-version-config-file
+        io/resource
+        slurp
+        str/trim)
+    (catch Exception e
+      (error "Failed to parse preview gem UMM version, will use the default version:"
+             default-preview-gem-umm-version
+             (.printStackTrace e)))))
```

Comment:
I'm not sure this is the right thing to do. The config file SHOULD be correct. If it is not then something is wrong and you should not just swallow this exception. This will allow things to start up with possibly the wrong gem version.